salt/journal/critical-css.sls

```diff
@@ -49,17 +49,26 @@ journal-local-demo-parameters:
         - require:
             - journal-local-demo-separate-folder
 
+journal-local-demo-critical-css:
```

Comment:
Temporary, a better solution is to make the folder exist in the repo with a `.gitkeep`. (Twig fails if it doesn't exist.)